test/fuzz/test_runner.py

```diff
@@ -182,6 +183,11 @@ def main():
             use_valgrind=args.valgrind,
         )
 
+        shutil_disk_usage_stat_ = shutil.disk_usage(args.corpus_dir)
+        if shutil_disk_usage_stat_.free < 1:
+            logging.debug("No space left on device")
+            sys.exit(-1)
```

Comment:
Is this needed? Pretty sure libFuzzer will print this error by itself.

Also, the fuzz inputs usually do not reside in the `/tmp` dir, which was the motivation for this patch?